platforms/documentation/docs/src/docs/userguide/dep-man/02-declaring-dependency-versions/single_versions.adoc

```diff
@@ -47,7 +47,7 @@ Gradle supports different ways of declaring a version string:
 
 When the upper or lower bound is missing, the range has no upper or lower bound.
 
-The symbol `]` can be used instead of `(` for an exclusive lower bound and `[` instead of `)` for an exclusive upper bound. e.g. `]1.0, 2.0[`.
+The symbol `]` can be used instead of `(` for an exclusive lower bound and `[` instead of `)` for an exclusive upper bound. e.g. `]1.0, 2.0[`. Note: The range `[1.0,[` is not appropriate, as it lacks an upper bound, and can be rewritten as `[1.0,)`.
```

Comment:
```suggestion
The symbol `]` can be used instead of `(` for an exclusive lower bound and `[` instead of `)` for an exclusive upper bound. e.g. `]1.0, 2.0[`. 

NOTE: The range `[1.0,[` is not appropriate, as it lacks an upper bound and should be rewritten as `[1.0,)`.
```